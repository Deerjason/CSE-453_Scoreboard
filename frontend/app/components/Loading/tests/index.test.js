import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../index';

test('Loading Component', () => {
  const component = renderer.create(<Loading />);
  expect(component.toJSON()).toMatchSnapshot();
});
