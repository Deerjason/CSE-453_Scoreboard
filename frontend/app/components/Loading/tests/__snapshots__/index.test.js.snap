// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component 1`] = `
<div
  className="MuiCircularProgress-root Loading-circularProgress-1 MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
  role="progressbar"
  style={
    Object {
      "height": 90,
      "width": 90,
    }
  }
>
  <svg
    className="MuiCircularProgress-svg"
    viewBox="22 22 44 44"
  >
    <circle
      className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
      cx={44}
      cy={44}
      fill="none"
      r={21.5}
      strokeWidth={1}
      style={Object {}}
    />
  </svg>
</div>
`;
